### DISCUSSION TITLE
Not Starting in Macbook pro M1 macOS Monterey.

### DISCUSSION THREAD

**Discussion by barefootitalian59** [utmapp/UTM#6797] [UTM Help (macOS)]
Hi, 
And thanks for the software. 
I'm on a Macbook pro M1 16GB and, although I installed UTM successfully on an intel Monterey, on this M1 (Monterey) I always get the following screen and the startup never happens...(PS: I followed the installation procedure).
Can anyone help please?
Thanks!
![Screen Shot 2024-10-14 at 1 52 37 pm](https://github.com/user-attachments/assets/7a5510f5-8113-44b7-b958-1664c4a184d8)

**Comment by barefootitalian59** [comment]
PROBLEM SOLVED! All my fault.........

**Comment by 6eanut** [comment]
I guess it's because amd iso was used in emulation
